modules/client-configuration/pages/clients-ubuntu.adoc

```diff
@@ -20,111 +20,129 @@ However, the root user on {ubuntu} is disabled by default, so to use bootstrappi
 
 
 
-== Prepare to Register
+== Add Software Channels
 
-Some preparation is required before you can register {ubuntu} clients to the {productname} Server.
+Before you register {ubuntu} clients to your {productname} Server, check that you have the {ubuntu} product enabled, and the required channels are fully synchronized.
 
 
 ifeval::[{suma-content} == true]
-.Procedure: Adding Client Tools Channels
 
-Before you begin, ensure you have the {ubuntu} product enabled, and have synchronized the {ubuntu} channels for {scc}:
+The channels you need for this procedure are:
 
-You can do this from {webui} or using command prompt, at your choice.
+[[ubuntu-channels-wizard]]
+[cols="1,1,1,1,1", options="header"]
+.Ubuntu Channels - WebUI
+|===
+| OS Version
+| Base Channel
+| Security Channel
+| Updates Channel
+| Universe Channel
 
-For {ubuntu} 20.04:
+| {ubuntu} 20.04
+| Ubuntu 20.04
+| ubuntu-2004-amd64-main-security for amd64
+| ubuntu-2004-amd64-main-updates for amd64
+| ubuntu-2004-amd64-universe for amd64
 
-* From the {webui}, add [systemitem]``Ubuntu 20.04`` and [systemitem]``SUSE Linux Enterprise Client Tools Ubuntu 2004 amd64``.
-* From the command prompt, add [systemitem]``ubuntu-20.04-pool-amd64`` and [systemitem]``ubuntu-20.04-suse-manager-tools-amd64``.
+|===
 
-In the {productname} {webui}, navigate to menu:Software[Channel List > All].
-Verify that you have a base channel and a child channel for your architecture.
+include::snippets/add_channels_wizard.adoc[]
 
-For example, for {ubuntu} 20.04:
+endif::[]
 
-* Base channel: [systemitem]``ubuntu-2004-amd64-main for amd64``
-* Child channel: [systemitem]``Ubuntu-20.04-SUSE-Manager-Tools for amd64``
 
+ifeval::[{suma-content} == true]
+The channels you need for this procedure are:
 
-ifeval::[{uyuni-content} == true]
-.Procedure: Adding the {ubuntu} Channels
+[[ubuntu-channels-cli-suma]]
+[cols="1,1,1", options="header"]
+.Ubuntu Channels - CLI
+|===
 
-. At the command prompt on the {productname} Server, as root, add the {ubuntu} channels.
+| OS Version
+| Base Channel
+| Tools Channel
 
-For {ubuntu} 20.04:
-----
-spacewalk-common-channels \
-ubuntu-2004-pool-amd64-uyuni \
-ubuntu-2004-amd64-main-uyuni \
-ubuntu-2004-amd64-main-updates-uyuni \
-ubuntu-2004-amd64-main-security-uyuni \
-ubuntu-2004-amd64-universe-uyuni \
-ubuntu-2004-amd64-universe-updates-uyuni \
-ubuntu-2004-amd64-uyuni-client
-----
+| {ubuntu} 20.04
+| ubuntu-20.04-pool-amd64
```

Comment:
No, the base channel is now the "real" ubuntu base channel `ubuntu-2004-amd64-main for amd64` [ubuntu-2004-amd64-main-amd64]